apidoc/Titanium/UI/Window.yml

```diff
@@ -535,15 +535,6 @@ methods:
         default: "{ animated: true }"
     platforms: [iphone, ipad, macos]
 
-  - name: hideTabBar
-    summary: Hides the tab bar. Must be called before opening the window.
-    description: |
-        To hide the tab bar when opening a window as a child of a tab, call
-        `hideTabBar` or set `tabBarHidden` to `true` **before** opening the window.
-
-        If the window is not a child of a tab, this method has no effect.
-    platforms: [iphone, ipad, macos]
```

Comment:
This can't just be removed. Please find a way to either make it backwards compatible + warning log or keep it and add an error log that it's not available anymore. But then, we cannot release it before Ti SDK 13.0.0

Reply:
You're right as I checked it again and found it working fine to temporary hide the TabBar per window basis, except not effective on elevated tab-bar on iPad, which is fine as elevated tab-bar can be tackled via new additions in this PR. I will revert this property for Window.